src/devices/things/ITemperatureSensor.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device
+{
+    /// <summary>
+    /// Abstraction for temperature
+    /// </summary>
+    public interface ITemperatureSensor
```

Comment:
or maybe just ITemperature

Reply:
This could have static helper methods to convert between Fahrenheit, Celsius, Kelvin, etc.

Reply:
Same for meter to imperial system.

Reply:
@shaggygi I agree we should have them - let's not do it in this PR though - once we get the design for the low level sensors we can propose those and figure out which library should they be at and what should be the namespace